src/main/kotlin/com/github/lppedd/cc/editor/CommitTabActionConfigurationCustomizer.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.lppedd.cc.editor
+
+import com.intellij.ide.plugins.PluginManager
+import com.intellij.openapi.actionSystem.ActionManager
+import com.intellij.openapi.actionSystem.Constraints
+import com.intellij.openapi.actionSystem.DefaultActionGroup
+import com.intellij.openapi.actionSystem.impl.ActionConfigurationCustomizer
+import com.intellij.openapi.extensions.PluginId
+
+/**
+ * @author Edoardo Luppi
+ */
+internal class CommitTabActionConfigurationCustomizer : ActionConfigurationCustomizer {
+  private val actionId = "EditorTab"
+  private val groupId = "EditorActions"
+
+  override fun customize(actionManager: ActionManager) {
+    // Rider register its own EditorTab action, see com.jetbrains.rider.editorActions.FrontendTabAction.
+    // Unfortunately that action is coded in Kotlin and marked as final,
+    // so the only way to avoid breaking Rider is simply to not offer
+    // enhanced tabbing in the commit dialog
```

Comment:
FYI, we can make it not `final` in Rider core if it's beneficial for the purposes of the plugin. I'm still not sure if it will work properly, but we can try.

Could you please explain what you would like to do within the action?

Reply:
@ForNeVeR the override of the EditorTab allows a sort-of structural navigation. See GIF, I've highlighter the usages of tab.   
Default IJ tabbing simply moves the focus to the next dialog field. My plan at the beginning was to test this feature out but I never found the time to improve it, as ideally I should allow moving between all the different message tokens.

![tabbing](https://user-images.githubusercontent.com/19871649/226175498-719d6cf0-9264-4c7e-b0bd-b19486b2c7dd.gif)

Reply:
See the Javadoc for the class [CommitTabAction](https://github.com/lppedd/idea-conventional-commit/blob/master/src/main/kotlin/com/github/lppedd/cc/editor/CommitTabAction.kt#L18).